### DISCUSSION TITLE
how to turn reasct native app ещ landscape page

### DISCUSSION THREAD

**Discussion by riaron777** [react-native-community/discussions-and-proposals#743] [Feedback]
why next code dos'nt worked?

import React,{useState} from 'react';
import {Text, StyleSheet, View} from 'react-native';

export default function App(){
return(
    <View style={styles.baseText}>
            <Text>hello world</Text>
     </View>
);

}

const styles = StyleSheet.create({
    baseText: {
        textAlign: 'center', // <-- the magic
        fontWeight: 'bold',
        fontSize: 18,
        marginTop: 0,
        width: 200,
        backgroundColor: 'yellow',
        transform: [{scaleX: "-1"}]
    }
});

**Comment by cortinico** [comment]
Hi,
This repo is reserved to long conversation. You should seek support to StackOverflow for those kind of problems
